admin-dev/themes/new-theme/js/vue/pages/product/index.js

```diff
@@ -0,0 +1,9 @@
+import Vue from 'vue';
+import ProductImageUpload from './ProductImageUpload';
+
+export default new Vue({
+  el: '#productPageVue',
```

Comment:
Would be `.image-dropdown` as you can have multiple dropdown on the same page (not in the product page but just make it much reusable as possible)